features/step_definitions/sriov.rb

```diff
@@ -116,6 +116,21 @@
     teardown_add {
       step %Q/I delete the "#{cr_name}" sriovnetwork/
     }
+  when "sriovibnetwork"
+    cr_project = opts[:project]
+    logger.info("resource type is: sriovibnetwork")
+    if sriov_i_b_network(cr_name).exists?
+       step %Q/I delete the "#{cr_name}" sriovibnetwork/
+    end
+
+    step %Q/I run oc create over "#{resource_yaml}" replacing paths:/,table(%{
+      | ["spec"]["networkNamespace"] | "#{cr_project}" |
+    })
+
+    step %Q{the step should succeed}
+    teardown_add {
+      step %Q/I delete the "#{cr_name}" sriovibnetwork/
+    }
```

Comment:
Can be simply with `step %Q/admin ensures "xxx_resource_name_xxx" yyy_resource_type_yyy is deleted from the "zzz_project_name_zzz" project after scenario/`

Reply:
updated. @liangxia  
Could you help take a look, again.